Lib/glyphsLib/classes.py

```diff
@@ -534,6 +534,8 @@ def values(self):
 
     def append(self, FontMaster):
         FontMaster.font = self._owner
+        if FontMaster.id and len(FontMaster.id) > 0 and self[FontMaster.id]:
```

Comment:
Why do you do an extra check of the `len`? It looks like `if FontMaster.id` will already be False for the empty string?

```suggestion
        if FontMaster.id and self[FontMaster.id]:
```